_data/projects.yml

```diff
@@ -403,7 +403,7 @@
 - name: "NDCube"
   description: "A Python package for manipulating, inspecting and visualizing multi-dimensional contiguous and non-contiguous coordinate-aware data arrays"
   logo: "https://raw.githubusercontent.com/sunpy/ndcube/master/docs/logo/ndcube.png"
-  docs: "https://docs.sunpy.org/projects/ndcube/en/stable/"
+  docs: "https://docs.sunpy.org/projects/ndcube/"
```

Comment:
This should be set on the RTD side to ensure the base url goes to the desired version and language.

Reply:
Sorry, but what does "RTD side" stand for? It looks like you made this modification to many projects (and removed some trailing slashes). Can you please clarify why? If you have good arguments for doing so I can accept it, but we don't usually let people modify other peoples' project entries.

Reply:
I am happy to undo the changes. 

What I meant (with my poorly worded text) is that on the read the docs configuration for a project, you set the default version that the base url will load for a user when they access it. It will also (if there is one), change to a translated version if one exists for the user's system language. Therefore the url for a read the docs link should be the base url as I did for these.

Reply:
Do you know if that default version is a required configuration? Is it possible that one of these projects hasn't set a default version, and could that negatively affect them? It's been a while since I made something with read the docs and I don't remember. 

I'm for these changes in principle, not forcing the language to be English in particular.

Reply:
The issue is that the default is set on the RTD side and that's the main branch, not the stable one. 
So unless someone went into the settings and changed it to stable, the base url will point to latest. 

E.g, for ndcube, https://docs.sunpy.org/projects/ndcube -> https://docs.sunpy.org/projects/ndcube/en/stable/
While for speasy, the url here was fixed to stable, but the base url now will go to latest instead.

The trailing slash removal is unnecessary, I just prefer it not being there since it makes it more clear what the end of the URL is. That can be added back.

Reply:
I understand. It's a reasonable change, I'm in favor of keeping it. I highly doubt anyone will complain or even notice, but I'll know what to revert if I have to.